tests/helpers/perf_run_helper.rb

```diff
@@ -787,6 +805,15 @@ def get_perf_result
       perf.log_csv
       copy_archive_files
     end
+    scp_to(master, "util/metrics/collect_metrics_files.rb", "/root/collect_metrics_files.rb")
+    on(master, "ruby /root/collect_metrics_files.rb")
+    start_epoch = File.read("#{@archive_root}/start_epoch")
+    end_epoch = File.read("#{@archive_root}/end_epoch")
+    cmf_output = on(master,
+                    "ruby /root/collect_metrics_files.rb --start_epoch #{start_epoch} --end_epoch #{end_epoch}")
```

Comment:
Is ruby in the PATH? Is it the expected ruby?  Should this be set to "env PATH=\"#{master['privatebindir']}:${PATH}\" ruby ..."?

Reply:
Since we didn't install ruby on the master as far as I know.  I assume it is part of the puppet install process. I could change the path though just to be on the safe side.

Reply:
and you meant export not env right?
like
```
cmf_output = on(master,
                    "export PATH=\"#{master['privatebindir']}:${PATH}\";" +
                    " ruby /root/collect_metrics_files.rb --start_epoch #{start_epoch} --end_epoch #{end_epoch}")
                 .output
```

Reply:
I meant env in order for it to be available for the life of the command not the life of the session.

Reply:
got it sorted an updated